conf/msg_conf/map_msg.conf

```
@@ -11,6 +11,8 @@
 // 270: *%s %s* (@me format)
 // 270: *%.0s%s* (remove the character's name from it)
 
+// Modify MAP_MAX_MSG in src/map/map.cpp to increase the maximum message limit.
```

```suggestion
// Define a custom maximum value for MAP_MAX_MSG in /src/custom/defines_pre.hpp to increase the maximum message limit.
```
As correctly suggested by @Daegaladh